weave/trace_server/opentelemetry/constants.py

```diff
@@ -0,0 +1,56 @@
+INPUT_KEYS = {
```

Comment:
I would document this file - this seems to be your ultimate source of truth for mapping

Reply:
Lets merge in https://github.com/wandb/weave/pull/4181 and https://github.com/wandb/weave/pull/4182/files since it changes some stuff and then I'll document it